docs/08-developer-guide/01-build/03-docker-images.md

```diff
@@ -2,10 +2,126 @@
 sidebar_label: Docker Images
 ---
 
+<!-- cspell:word testkrb5 -->
+
 # Building Ozone Docker Images
 
-**TODO:** File a subtask under [HDDS-9861](https://issues.apache.org/jira/browse/HDDS-9861) and complete this page or section.
+This page provides an overview of the Docker images maintained by the Apache Ozone community for developing and testing Ozone.  It also describes the workflow to be followed when making changes to one of these images.
+
+## Images
+
+### Base Image (without any Ozone)
```

Comment:
IMO we should use the image names as the headers and explain them in the paragraph, since we usually refer to them by name. Right now the headers are sort of explanations as well, and the actual image name is at the beginning of the paragraph.

Reply:
Makes sense, updated.